server/spec/requests/auth_request_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe "Authentication", type: :request do
+  describe "POST /register" do
+    let(:user_params) { attributes_for(:user) }
```

Comment:
What does `attributes_for` do?

Reply:
Instead of returning a `User` object it returns a hash with all the parameters that you would use to create a `User` object

Reply:
Check out this for reference

https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#using-factories